internal/component/prometheus/write/queue/component.go

```diff
@@ -113,7 +110,7 @@ func (s *Queue) Update(args component.Arguments) error {
 		if err != nil {
 			return err
 		}
-		end.Start()
+		end.Start(s.ctx)
```

Comment:
This will be nil if Update is called before Run. Will passing nil to Start cause issues?